pagopa/pn-frontend-e2e
# PN-9417-e-2-e-destinatario-accesso-alla-sezione-notifiche-da-parte-delegato

### Issue Description
The **PN-9417** is failing because it doesn't test the correct behaviour described in the task: [PN-9417](https://pagopa.atlassian.net/browse/PN-9417)

### Changes Made:
- adjusted some feature steps
- added **pull request template**

### How to Test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_PFdelegatovisualizzaDettaglioNotifica` and verify if the test works fine.
Also a run into AWS CodeBuild is necessary in order to test everything.

[PN-9417]: https://pagopa.atlassian.net/browse/PN-9417?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
